src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1632,7 +1633,25 @@ function ($a) {
                         ->arrayNode('stop_worker_on_signals')
                             ->defaultValue([])
                             ->info('A list of signals that should stop the worker; defaults to SIGTERM and SIGINT.')
-                            ->integerPrototype()->end()
+                            ->beforeNormalization()
+                                ->always()
```

Comment:
Once the weird handling of the array is removed, this could become `ifString()` instead as that's the only case that needs a change on the value.

Reply:
Well actually as we're receiving an array, I can't use `ifString()`. Or maybe I'm misunderstanding how to use it?